tools/src/main/resources/frontendconfigs/build.sh

```diff
@@ -5,24 +5,27 @@ echo "Cleaning up previous build artifacts ..."
 rm -rf frontend
 # Assemble assets
 echo "Assembling assets ..."
-npx --legacy-peer-deps openmrs@5.8.1 assemble \
+npx --legacy-peer-deps openmrs@latest assemble \
```

Comment:
@slubwama i think lets leave this at a version than latest

Reply:
@akileng56  that would be hard to build for anyone without the selected version in their local npm. They have to download it manually